### ISSUE TITLE
Toolbar Run Icon

### ISSUE THREAD

**Issue by felipefoz** [JKISoftware/Caraya#97]
I know there are a lot of ways of running tests including quick drop plugins.
But it would be interesting to add an icon to the toolbar to run the tests.
I will myself add this eventually, but nowadays I have no copy or way to install LV 2013.
Do you think it would be interesting?
Regards,

**Comment by jimkring** [comment]
That's a great idea @felipefoz. We could add to the LabVIEW project toolbar pretty easily.

_(side note: adding it's to the VI toolbar isn't a supported LabVIEW IDE feature/extension point)._

**Comment by felipefoz** [comment]
Yep, there is that. But I think just the "Run Tests in Active Project VI" would do no harm, would it? =P

**Comment by jimkring** [comment]
Can you share a screenshot of where you would put this button? Would it...

1. Run all the tests in the active project?
2. Run all tests that test the active VI?
3. Run all tests inside the active VI?

**Comment by felipefoz** [comment]
Currently, I would go for option one only, but the second option seems also interesting, but only if it is already implemented, otherwise I believe it is asking too much.

![Caraya_toolbar](https://user-images.githubusercontent.com/63814202/83190408-c1b69480-a108-11ea-9244-71d89a7735b2.PNG)

**Comment by jimkring** [comment]
Thanks for clarifying that. Yes, that's pretty doable and I think it would be a nice feature people would use often.

**Comment by felipefoz** [comment]
I appreciate the fast feedback and let me know if you need anything.
